pagopa/pn-frontend
# Fix(pn-12975) Fix menu in virtualKeysTable

## Short description
change how handle menu in virtualContextMenu

## List of changes proposed in this pull request

- inserted function that returns how many conditions are respected
- check on the length of the conditions


## How to test
for testing have 3 virtual key for giuseppe ungaretti and with Dante ALighieri user. You should see no menu for active virtual, as no actions are available

this PR are depend of https://github.com/pagopa/pn-frontend/pull/1403
